astroplan/utils.py

```diff
@@ -92,9 +92,16 @@ def _get_IERS_A_table(warn_update=14*u.day):
     if IERS_A_in_cache():
         table = iers.IERS_Auto.open()
         # Use polar motion flag to identify last observation before predictions
-        index_of_last_observation = ''.join(table['PolPMFlag_A']).index('IP')
-        time_of_last_observation = Time(table['MJD'][index_of_last_observation],
-                                        format='mjd')
+        if 'PolPMFlag_A' in table.colnames:
+            index_of_last_observation = ''.join(table['PolPMFlag_A']).index('IP')
+            time_of_last_observation = Time(table['MJD'][index_of_last_observation],
+                                            format='mjd')
+            
+        # If time of last observation is not available, set it equal to the
+        # final prediction in the table:
```

Comment:
might be good to add a warning here stating that IERS_A is not available and older data is being used instead. maybe include the date of the last observation in the warning message.

Reply:
In the interest of preventing more IERS warnings from popping up, which are already a significant source of headaches for people, I think I might let this one go by silently...? I want to move as much of the IERS responsibilities into astropy as possible. Does that sound ok?

Reply:
If you have strong feelings, please open a new issue, I'm merging this one for now.

Reply:
this is great, thanks! i double-checked and astropy should indeed be throwing its own warning in this case so no need for redundancy.